### PR TITLE
Certificate validation

### LINKED ISSUE #34: Resolve Suggestions According to ToB Report 6, MAY 2024
### Description

3\. Missing certificate validation (Med)
4\. Synchronous function calls inside asynchronous functions (Info)
5\. Hardcoded credentials (Low)
6\. Use of outdated libraries (Info)
7\. Stack traces in Express are not disabled (Info)
8\. Docker Compose ports exposed on all interfaces (Info)

For Item 7, this can be resolved by adding flag in AWS deployment script, which is not included in this Repo

### PR DESCRIPTION
**Pull request by Phanco**
### What was the problem?

This PR resolves #34, Item 3

### How was it solved?
- Deployer need to pass a certificate in order to start psql with SSL mode

### How was it tested?
- It runs successfully in local connecting to DB in Digitalocean with cert and login credentials

### CODE REVIEW

File: packages/claim-backend/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Claim Backend
 
-This package supplies Leaf details to frontend by providing an endpoint that responds with details necessary for the claim. It also stores partial signatures of claims from multisig addresses. Details of Leaf can be obtained from [Tech Design](../../documentation/Tech_Design.md).
+This package supplies Leaf details to frontend by providing an endpoint that responds with details necessary for the
+claim. It also stores partial signatures of claims from multisig addresses. Details of Leaf can be obtained
+from [Tech Design](../../documentation/Tech_Design.md).
```
Comment (matjazv) [inline packages/claim-backend/README.md]: Those lines seem to be identical just some break lines were added?

File: packages/claim-backend/README.md [hunk 2]
```diff
@@ -11,19 +13,20 @@ Apart from Node version 18, Claim Backend also requires:
...
+| ---------------- | ---------------------------------------------------------- | ------------------------- |
+| BACKEND_HOST     | IP used by backend (Default: 127.0.0.1)                    | false                     |
+| BACKEND_PORT     | Port used by backend (Default: 3000)                       | false                     |
+| DB_PORT          | Port used by PostgreSQL DB (Default: 5432)                 | false                     |
+| DB_HOST          | Host used by PostgreSQL DB (Default: 127.0.0.1)            | false                     |
+| DB_DATABASE      | Name of Database of PostgreSQL DB (Default: claim-backend) | false                     |
+| DB_USERNAME      | Username of PostgreSQL DB (Default: claim-backend)         | false                     |
+| DB_PASSWORD      | Password of Database of PostgreSQL DB (Default: passwd)    | false                     |
+| DB_SSLMODE       | A flag to turn on SSL Connection on DB (Default: false)    | false                     |
+| DB_SSL_CERT_PATH | Path to SSL Certificate(.crt or .pem) (Default: `null`)    | When `DB_SSLMODE` is True |
```
Comment (matjazv) [inline packages/claim-backend/README.md]:
```suggestion
| DB_SSL_CERT_PATH | Path to SSL Certificate (.crt or .pem) (Default: `null`)    | When `DB_SSLMODE` is True |
```

File: packages/claim-backend/README.md [hunk 3]
```diff
@@ -39,9 +42,12 @@ $ yarn server
...
 ### `/checkEligibility`
 
-It accepts POST request with param `lskAddress`. If the address is eligible for claiming some LSK, it will return the amount of tokens (under `account`), proof and, should this address match to any public key of a multisignature account, it will also be displayed here (under `multisigAccounts`).
+It accepts POST request with param `lskAddress`. If the address is eligible for claiming some LSK, it will return the
+amount of tokens (under `account`), proof and, should this address match to any public key of a multisignature account,
+it will also be displayed here (under `multisigAccounts`).
 
-`signatures` field stores valid signatures of addresses in multiAccounts, which are previously submitted by `/submitMultisig`.
+`signatures` field stores valid signatures of addresses in multiAccounts, which are previously submitted
+by `/submitMultisig`.
```
Comment (matjazv) [inline packages/claim-backend/README.md]: Those lines are also identical. Better to revert them to have less changes inside files.

File: packages/claim-backend/README.md [hunk 4]
```diff
@@ -100,7 +106,8 @@ curl --location 'http://127.0.0.1:3000/rpc' \
 
 ### `/submitMultisig`
 
-After the user has signed a claim message for a multisignature account, this API will be called to store the signature at the backend DB.
+After the user has signed a claim message for a multisignature account, this API will be called to store the signature
+at the backend DB.
```
Comment (matjazv) [inline packages/claim-backend/README.md]: Same is true for those lines.

**Review by matjazv — CHANGES_REQUESTED** [review]

File: packages/claim-backend/README.md (hunk 1 above)
Comment (Phanco) [reply packages/claim-backend/README.md]: updated

File: packages/claim-backend/README.md (hunk 2 above)
Comment (Phanco) [reply packages/claim-backend/README.md]: updated

File: packages/claim-backend/README.md (hunk 3 above)
Comment (Phanco) [reply packages/claim-backend/README.md]: updated

File: packages/claim-backend/README.md (hunk 4 above)
Comment (Phanco) [reply packages/claim-backend/README.md]: updated

**Review by matjazv — APPROVED** [review]

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**